docs/learn/learn-proxies.md

```diff
@@ -208,20 +220,38 @@ and the `ProxyDepositFactor` is
 We can add a layer of security to proxies by giving them a delay time. The delay will be quantified
 in blocks. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} has approximately 6
 seconds of block time. A delay value of 10 will mean ten blocks, which equals about one minute
-delay. The proxy will announce its intended action and will wait for the number of blocks defined in
-the delay time before executing it. The proxy will include the hash of the intended function call in
-the announcement. Within this time window, the intended action may be canceled by accounts that
-control the proxy. Now we can use proxies knowing that any malicious actions can be noticed and
-reverted within a delay period.
+delay. The proxy will announce its intended action using the `proxy.announce` extrinsic and will
+wait for the number of blocks defined in the delay time before executing it. The proxy will include
+the hash of the intended function call in the announcement. Within this time window, the intended
+action may be canceled by accounts that control the proxy. This can be done by the proxy itself
+using the `proxy.removeAnnouncement` extrinsic or by the proxied account using the the
+`proxy.rejectAnnouncement` extrinsic. Now we can use proxies knowing that any malicious actions can
+be noticed and reverted within a delay period. After the time-delay, the proxy can use the
+`proxy.proxyAnnounced` extrinsic to execute the announced call.
+
+:::info
+
+See [this video tutorial](https://youtu.be/LxIW7t15Iuw) to learn how you can setup and use
+time-delayed proxies. The video goes through the example below.
+
+:::
+
+Let's take for example the stash account Eleanor that has a controller Charly. Eleanor does not
+fully trust Charly, and as a consequence sets him as a time-delayed staking proxy. In this way, if
+Charly submits an extrinsic to change the controller to Bob, such extrinsic can be rejected by
+Eleanor. Bob can be even more malicious than Charly and change the reward destination to another
+account he only controls. Remember that Eleanor added Charly as proxy, but she might not want to add
+Bob as a controller. This implies that Eleanor monitors Charly, and that within the time-delay she
+can spot the announced extrinsic. As a monitor tool you can use a block explorer.
```

Comment:
You need not use a block explorer. This information can be queried on-chain directly.

```suggestion
can spot the announced extrinsic. Eleanor can check all the proxy call announcements made on her account's behalf on-chain. On Polkadot-JS UI, go to Developer > Storage > Proxy > Announcements to view the list of unexecuted proxy calls on Eleanor's account.
```